spark/client/conf.go

```diff
@@ -0,0 +1,127 @@
+package client
+
+import (
+	"context"
+
+	proto "github.com/apache/spark-connect-go/v35/internal/generated"
+	"github.com/apache/spark-connect-go/v35/spark/client/base"
+)
+
+// Public interface RuntimeConfig
+type RuntimeConfig interface {
+	GetAll(ctx context.Context) (*map[string]string, error)
```

Comment:
no need to return a pointer to a map

Reply:
I agree, it is ugly. I am used to work with pointers but it leads to ugly user code such as (*map)["key"] which is really not bringing any value, so I will adjust